client/src/components/DoughnutChart.js

```diff
@@ -0,0 +1,44 @@
+import { Doughnut } from "react-chartjs-2";
+import propTypes from "prop-types";
```

Comment:
as a convention, the propTypes that import from the "prop-types" should be PropTypes
(The first Letter is Capital )

please see the [link](https://reactjs.org/docs/typechecking-with-proptypes.html)